package.json

```diff
@@ -26,7 +26,7 @@
   "version": "1.0.34",
   "private": false,
   "engines": {
-    "node": ">=7.5.0"
+    "node": ">=18.0.0"
```

Comment:
using functions in `node:fs` and `await`, which require a recent versions of node